include/zephyr/net/wifi_mgmt.h

```diff
@@ -1168,6 +1200,21 @@ struct wifi_mgmt_ops {
 	 * @return 0 if ok, < 0 if error
 	 */
 	int (*dpp_dispatch)(const struct device *dev, struct wifi_dpp_params *params);
+	/** Start wps pbc
+	 *
+	 * @param dev Pointer to the device structure for the driver instance
+	 *
+	 * @return 0 if ok, < 0 if error
+	 */
+	int (*wps_pbc)(const struct device *dev);
+	/** Set or get wps pin
+	 *
+	 * @param dev Pointer to the device structure for the driver instance
+	 * @param params wps pin operarion and pin number
+	 *
+	 * @return 0 if ok, < 0 if error
+	 */
+	int (*wps_pin)(const struct device *dev, struct wifi_wps_pin_params *params);
 };
```

Comment:
do we really need two different APIs? can't we merge them as `wps_config` and extend `params` ?